packages/eui/src/components/steps/step_number.tsx

```diff
@@ -51,7 +52,7 @@ export interface EuiStepNumberProps
   status?: EuiStepStatus;
   number?: number;
   /**
-   * Title sizing equivalent to EuiTitle, but only `m`, `s` and `xs`. Defaults to `s`
+   * Title sizing equivalent to EuiTitle, but only `m`, `s`, `xs` and `xxs`. Defaults to `s`
    */
   titleSize?: EuiStepProps['titleSize'];
```

Comment:
Ooooo. For the **EuiStepNumber** component specifically I'm SO tempted to have you do this:

```suggestion
   * Title sizing equivalent to EuiTitle, but only `m`, `s`, `xs`.
   * `none` indicates no step number should be rendered.
   * @default s
   */
  titleSize?: EuiStepProps['titleSize'] | 'none';
```

Reply:
I do like this idea for `EuiStepNumber`, it definitely makes sense! But `none` would not make sense for the parents `EuiSteps` and `EuiStepsHorizontal` as it's a combination of title and number size and `titleSize/size="none"` seems confusing there.
I think it's still valid to update this way but `none` needs to be mapped to `xxs` and `xs` respectively, I'd say. 

Random thought: I think `titleSize` doesn't really make sense as naming for this prop - imho just `size` would fit here better 😅

Reply:
Renamed the `xxs` size on `EuiStepNumber` to `none` in this [commit](https://github.com/elastic/eui/pull/7813/commits/712476c576fa247bec23235cdc0acfc96a5c0732).

Updated the wrapper components to map to `none` in this [commit](https://github.com/elastic/eui/pull/7813/commits/0de2968c7eb011083111aee2e5671cf821f57e47).

Reply:
Hooray! Yes, the `none` suggestion was only for `EuiStepNumber` specifically, not for the parents - `xxs` still makes more sense there. Awesome work with the change!

And yeah I similarly wish `EuiSteps` just had `size` instead of `titleSize` 🫠